src/graph/visitor/DeducePropsVisitor.cpp

```diff
@@ -173,17 +177,66 @@ void DeducePropsVisitor::visit(VersionedVariableExpression *expr) { reportError(
 
 void DeducePropsVisitor::visit(LabelExpression *expr) { reportError(expr); }
 
-void DeducePropsVisitor::visit(AttributeExpression *expr) { UNUSED(expr); }
-
 void DeducePropsVisitor::visit(LabelAttributeExpression *expr) { reportError(expr); }
 
 void DeducePropsVisitor::visit(ConstantExpression *expr) { UNUSED(expr); }
 
-void DeducePropsVisitor::visit(VertexExpression *expr) { UNUSED(expr); }
+void DeducePropsVisitor::visit(ColumnExpression *expr) { UNUSED(expr); }
 
-void DeducePropsVisitor::visit(EdgeExpression *expr) { UNUSED(expr); }
+void DeducePropsVisitor::visit(VertexExpression *expr) {
+  std::vector<TagID> tagIds;
+  if (tagIds_ == nullptr) {
```

Comment:
Initialize `tagIds_` in constructor. Here will call just when meet `VertexExpression`.

Reply:
fix